### PR TITLE
Don't use hardcoded colours

### LINKED ISSUE #47: Custom colors look inconsistent
The program uses custom colors (hand-picked for windows) for several elements rendering the look and feel very inconsistent with the Adwaita theme:
Screenshot from 2020-04-25 19-59-56

It's even worse with the Adwaita dark theme:
Screenshot from 2020-04-25 20-17-16

It worked well removing the custom colors on some of them but I'll have to find all of them and verify that there are no regressions on Windows.

### PR DESCRIPTION
**Pull request by mkroening**
Fixes #47.

This removes/replaces any colours which are hardcoded or incompatible with dark themes.

What do you think?

# Pictures

All pictures on Linux have been taken using GTK2.

Pictures on Windows have been taken with #49 merged.

### Adwaita
Before:
Screenshot from 2020-04-26 11-24-06
After:
Screenshot from 2020-04-26 11-27-51

### Adwaita-dark
Before:
Screenshot from 2020-04-26 11-26-47
After:
Screenshot from 2020-04-26 11-28-48

### Windows
Before:
Annotation 2020-04-26 113814
After:
Annotation 2020-04-26 111918

### Adwaita
Before:
Screenshot from 2020-04-26 11-25-59
After:
Screenshot from 2020-04-26 11-28-20

### Adwaita-dark
Before:
Screenshot from 2020-04-26 11-30-58
After:
Screenshot from 2020-04-26 11-29-14

### Windows
Before:
Screenshot_win10_2020-04-26_11:38:36
After:
Screenshot_win10_2020-04-26_11:36:46

### CODE REVIEW

File: src/view/MainFrame.cpp [hunk 1]
```diff
@@ -204,9 +203,9 @@ void MainFrame::handleLBSel(int index) {
 			list_tabs.at(i).second->SetBackgroundColour(wxSystemSettings::GetColour(wxSYS_COLOUR_HIGHLIGHT));
 			list_tabs.at(i).second->SetForegroundColour(wxSystemSettings::GetColour(wxSYS_COLOUR_HIGHLIGHTTEXT));
 		} else {
-			list_tabs.at(i).first->SetBackgroundColour(*wxWHITE);
-			list_tabs.at(i).second->SetBackgroundColour(*wxWHITE);
-			list_tabs.at(i).second->SetForegroundColour(*wxBLACK);
+			list_tabs.at(i).first->SetBackgroundColour(wxNullColour);
+			list_tabs.at(i).second->SetBackgroundColour(wxNullColour);
+			list_tabs.at(i).second->SetForegroundColour(wxNullColour);
```
Comment (yannikschaelte) [inline src/view/MainFrame.cpp]: is this still necessary then?

**Review by yannikschaelte — APPROVED** [review]
agreed. If I remember correctly, the reason to set the background color to white was to give it more of a "sketch paper" feeling, but of course this can easily mess things up, so agreed to follow the employed (color) theme.

File: src/view/MainFrame.cpp (hunk 1 above)
Comment (mkroening) [reply src/view/MainFrame.cpp]: This branch is necessary to remove the highlighting from the items again.

Thanks to your question I actually found that setting the background colour on `wxStaticText` does not seem to have any effect here. So I'll remove the second statement of each branch.

Comment (yannikschaelte) [reply src/view/MainFrame.cpp]: :+1:

**Pull request merged**